x/government/module.go

```diff
@@ -3,6 +3,10 @@ package government
 import (
 	"encoding/json"
 
+	"github.com/commercionetwork/commercionetwork/x/government/client/rest"
+
```

Comment:
```suggestion
```

Reply:
Uh? Should I remove that import? Without that, I cannot register rest routes though.

Reply:
just the empty line